src/app/components/new-board/new-board.component.ts

```diff
@@ -608,9 +476,10 @@ export class NewBoardComponent implements OnInit {
 
   // Save board data
   saveData() {
-    const boardTitle = document.getElementById('cb-title').innerHTML.trim();
-    const boardlData = document.getElementById('main-box').innerHTML.trim();
-    this.apiService.saveBoardData(boardTitle, boardlData);
+    // const boardTitle = document.getElementById('cb-title').innerHTML.trim();
```

Comment:
Why do we need to comment these lines here?

Reply:
The saveData function is resolved in the upcoming PR by @goliakshay357  using the Map object created in this PR to solve #171 and #192 . These lines will be rewritten as of now so to indicate their position it has been commented out as of now.